foo.ts

```diff
@@ -0,0 +1,29 @@
+import { SyncRedactor } from './src/index';
```

Comment:
Did you mean to check in this file? It looks like some test file.

Reply:
oops, nope